nservicebus/upgrades/5to6-moving-away-from-ibus.md

```diff
@@ -7,28 +7,38 @@ Starting with Version 6, the `IBus` interface has been deprecated.
 
 The `IBus` interface was one of the key interfaces when using previous versions of NServiceBus. It provided access to many of the operations like sending messages, subscribing to events, and manipulating headers. The `IBus` interface was available through dependency injection in message handlers, any custom components that were registered in the container and through other mechanisms like the saga base class.
 
+
+## Why was IBus deprecated?
```

Comment:
i know this approach is in other places but i dislike rhetorical questions as titles. how about "Reasons for IBus deprecated"